File: 3 - Implementacion/Adapter SDK/Java/v1.8/src/adapter/Log.java
package adapter;

/**
 * 
 * @author EA0734
 * Writes log to System.out
 */
public class Log {

	/**
	 * Writes a title to output
	 * @param message string to be printed
	 */
	public static void title(String message){
		System.out.println(message.toUpperCase());		
	}

	/**
	 * Writes an info to output
	 * @param message string to be printed
	 */
	public static void info(String message){
		System.out.println("INFO:" + message);		
	}

	/**
	 * Writes an error to output
	 * @param message string to be printed
	 */
	public static void error(String message){
		System.out.println("ERROR:" + message);		
	}
}

File: 3 - Implementacion/Adapter SDK/Net/v4.0/Log.cs
// -----------------------------------------------------------------------
// <copyright file="Log.cs" company="Prisma">
// Baufest (c) 2016
// </copyright>
// -----------------------------------------------------------------------

namespace Adapter
{
    using System;

    /// <summary>
    /// Provides Log to Console
    /// </summary>
    public class Log
    {
        /// <summary>
        /// Writes a error message with red color
        /// </summary>
        /// <param name="message">The message to write</param>
        public static void Error(string message)
        {
            Console.ForegroundColor = ConsoleColor.Red;
            Console.WriteLine(message);
            Console.ResetColor();
        }

        /// <summary>
        /// Writes a error message with white color
        /// </summary>
        /// <param name="message">The message to write</param>
        public static void Info(string message)
        {
            Console.ForegroundColor = ConsoleColor.Gray;
            Console.WriteLine(message);
            Console.ResetColor();
        }

        /// <summary>
        /// Writes a error message with white color
        /// </summary>
        /// <param name="message">The message to write</param>
        public static void Title(string message)
        {
            Console.ForegroundColor = ConsoleColor.White;
            Console.WriteLine(message);
            Console.ResetColor();
        }
    }
}
